src/block/view/index.js

```diff
@@ -3,13 +3,16 @@ export const View = ( props ) => {
 	const { className, url, aspectRatio } = props;
 	const embedURL = `https://www.figma.com/embed?embed_host=share&url=${ encodeURIComponent( url ) }`;
 	return (
-		<div className={ `${ className } is-${ aspectRatio }` }>
-			<iframe
-				key={ aspectRatio } /* force creation of a new iframe to force figma to update its layout */
-				src={ embedURL }
-				allowFullScreen
-				title="A figma file" /* TODO: figure out something more accessible */
-			/>
+		/* this extra wrapper div gets max-width set */
+		<div>
```

Comment:
Added an extra wrapping `div`. This `div` receives the `max-width` from the theme which the child `div` bases its aspect calculations off.

This might prevent `is-style-wide` from being applied via classes to break out of the `.entry-content` width. We might need to use a different classname.

Reply:
Have added new classnames for styling to combat the above.